kview/src/main/java/dev/ikm/komet/kview/mvvm/view/export/ExportController.java

```diff
@@ -277,8 +277,11 @@ void handleExport(ActionEvent exportEvent){
             fromDate = this.customFromEpochMillis == 0 ? transformStringInLocalDateTimeToEpochMillis(dateTimeFromLabel.getText()) : this.customFromEpochMillis;
             toDate = this.customToEpochMillis == 0 ? transformStringInLocalDateTimeToEpochMillis(dateTimeToLabel.getText()) : this.customToEpochMillis;
         }
-        String namePrefix = exportName.getText().equals("") ? "komet" : exportName.getText();
-        String initialFileName = "%s-%s".formatted(namePrefix, simpleDateFormat.format(new Date()));
+        // if the user enters a name then use that name, e.g. test.json or test.zip
+        // if the user does not enter a name, then default to komet-yyyyMMdd-HHmm.zip|.json
+        String initialFileName = exportName.getText().equals("")
```

Comment:
Can you try `isBlank()`? The user could accidentally enter a space.

Reply:
udpated